.github/workflows/release.yaml

```diff
@@ -1,10 +1,5 @@
 name: Release
 
-on:
-  push:
-    branches:
-      - main
-
```

Comment:
was this intentional?

Reply:
@JFrankfurt yes, zach suggested we disable automatic release until these redesign changes are done: https://www.notion.so/uniswaplabs/Widget-UI-Updates-Eng-Plan-deeec4509ca448b796ed41caf63b2bc3